examples/sample-app-vite/src/main.js

```diff
@@ -417,6 +417,8 @@ const questions2 = [
     type: "confirm",
     name: "confirm",
     message: "Are you sure?",
+    labelTrue: "Yes, I am sure!",
+    labelFalse: "No, I am still contemplating...",
```

Comment:
consider overriding just one of the labels to verify that the default values ​​are handled as expected.